### PR TITLE
🚛POST /transfer endpoint

### PR DESCRIPTION
**Pull request by gmkohler**
see commit messages.. planning to iterate

### CODE REVIEW

File: emoji/slack_emoji_client.py [hunk 1]
```diff
@@ -26,4 +29,21 @@ def remove_emoji(self, name):
...
-        return self.client.emoji_list().data['emoji']
+        if self._emoji_dict is None:
+            self._emoji_dict = self.client.emoji_list().data['emoji']
+
+        return self._emoji_dict
+
+    def hashes_for_emoji_images(self):
+        if self._hashes_for_emoji_images is None:
+            self._hashes_for_emoji_images = {}
+            destination_emoji_with_urls = { name: val for name, val in self.emoji_dict().items() if not val.startswith("alias:") }
```
Comment (clanam) [inline emoji/slack_emoji_client.py]: do we update this dict after more emoji are transferred?

File: emoji/emoji_transfer_service.py [hunk 2]
```diff
@@ -26,21 +26,11 @@ def collision_free_name(destination_dict, source_emoji_name, prefix="cav"):
 
     return destination_name
 
-def transfer(source_client, destination_client, source_emoji_name):
+def transfer(source_dict, destination_client, source_emoji_name):
```
Comment (clanam) [inline emoji/emoji_transfer_service.py]: i think https://github.com/gmkohler/emoji-exodus/blob/9484a99d020abd0c06edc265a6e306e889cab29f/application.py#L11 also needs to be updated to match this change in api

Comment (gmkohler) [reply emoji/emoji_transfer_service.py]: good catch

File: emoji/slack_emoji_client.py (hunk 1 above)
Comment (gmkohler) [reply emoji/slack_emoji_client.py]: No, support for multiple additions in the same request will be a new PR.  Still need to figure out the best way to do this.

**Pull request merged**